### ISSUE TITLE
marvel work  ,could you show more detail of loss_fn

### ISSUE THREAD

**Issue by gjpicker** [rafellerc/Pytorch-SiamFC#7]
I am confused with label_py.create_BCELogit_loss_label ,loss_py.BCELogit_Loss . we are yearning your loss_function's interpretation .

tanks for your maven

**Comment by rafellerc** [comment]
I'm currently writing a jupyter notebook with the full derivation, but in short, in section 2.2 the author uses the logistic loss **log(1+exp(-yv))**, where y are the labels, being -1 for negative (not center) and 1 for positive (center), and v are the outputs in the score map (the output of a convolutional layer without activation function, thus unbounded).

**Comment by rafellerc** [comment]
If you take the inverse of the expression inside the **log**, you get a sigmoid function (let's call it sigm()): **log(1+exp(-yv)) = -log(1/(1+exp(-yv))) = log(sigm(yv))**. Replacing the values of y we get: for y=1 -> **loss = -log(sigm(v))**, y=-1 -> **loss = -log(sigm(-v)) = -log(1 - sigm(v))**

**Comment by gjpicker** [comment]
thanks ,waiting intensely
